lib/lcformvalidation.d.ts

```diff
@@ -31,8 +31,8 @@
       isFormPristine(): boolean;
       validateFullForm(vm: any): Promise<FormValidationResult>;
       triggerFieldValidation(vm: any, key: string, value: any, filter?: any): Promise<FieldValidationResult>;
-      addFieldValidation(key : string, validation : (vm, value) => FieldValidationResult, filter? : any);
-      addFieldValidationAsync(key : string, validation : (vm, value) => Promise<FieldValidationResult>, filter? : any);
+      addFieldValidation(key : string, validation : (vm, value) => FieldValidationResult, filter? : any): IValidationEngine;
+      addFieldValidationAsync(key : string, validation : (vm, value) => Promise<FieldValidationResult>, filter? : any): IValidationEngine;
```

Comment:
That what's something we missed on @crsanti review, he didn't updated the typescript definitions. 

Lesson learned from this case:

- Avoid extra noise, just focus on the case.
- Migrate samples in order to use new features.